deebot_client/const.py

```diff
@@ -8,6 +8,7 @@
 PATH_API_PIM_PRODUCT_IOT_MAP = "pim/product/getProductIotMap"
 PATH_API_IOT_DEVMANAGER = "iot/devmanager.do"
 PATH_API_LG_LOG = "lg/log.do"
+PATH_API_DLN_LOG_LIST = "log/clean_result/list"
```

Comment:
```suggestion
_API_PREFIX = "api/"
PATH_API_APPSVR_APP = _API_PREFIX + "appsvr/app.do"
PATH_API_PIM_PRODUCT_IOT_MAP = _API_PREFIX + "pim/product/getProductIotMap"
PATH_API_IOT_DEVMANAGER = _API_PREFIX + "iot/devmanager.do"
PATH_API_LG_LOG = _API_PREFIX + "lg/log.do"
PATH_API_DLN_LOG_LIST = "app/dln/api/log/clean_result/list"
```

Please also adjust `PATH_API_APPSVR_APP` as github doesn't allow me to include it in this suggestion.

Imo this change is better than introducing changes to `deebot_client/authentication.py`

Reply:
Hope i understand the change correct.
I revert the changes in `authentication.py` and add instead prefix in `const.py`.
I have moved also the `users/user.do` api from `authentication.py` into `const.py` as it than needs also the prefix and to have all api's in one place.